### ISSUE TITLE
can't type anything to bito ai in intellij idea 2023.1-eap

### ISSUE THREAD

**Issue by MohammadMD1383** [gitbito/bitoai#1]
Hi, thanks for the great plugin.

I can't type anything to the bito ai text field after upgrading to latest version of the extension.

except that I can paste smth or send ctrl+enter or enter or backspace

**Comment by bitomukesh** [comment]
Thanks for reporting this. We are investigating this.

**Comment by nisha-bito** [comment]
Hi @MohammadMD1383 -
Can you help us with more details like Operating system you are using and Bito version. Any screenshot or a short video which can help us analyze and fix the issue on priority.

**Comment by MohammadMD1383** [comment]
```
Ubuntu 22.10
Intellij Idea Ultimate 2023.1-EAP
Bito AI 1.0.129
```


I can't type anything, just shortcuts like (ctrl+v, ...) work.

**Comment by nisha-bito** [comment]
Thanks for the information, we will look into the issue.

**Comment by nisha-bito** [comment]
hi @MohammadMD1383 ,

We analyzed and found that all flavor of Linux has the same issue on the version 2023.1. Where in any input box in plugin is not showing typed character while copy and paste works fine.  We have also validated this on few more IntelliJ plugins and same issue exists in all. 
We are working with JetBrains team for this issue.  And we shall update you with any progress on this. 

Thanks,
Nisha

**Comment by MohammadMD1383** [comment]
Thanks a lot!

**Comment by nisha-bito** [comment]
hi all,

we added 2 custmomer issues and 1 story in the change log https://bito.atlassian.net/wiki/spaces/EN/pages/224624641/Change+Log

  1.  Customer issue - https://bito.atlassian.net/browse/BITO-1939
  2.  Customer Issue - https://bito.atlassian.net/browse/BITO-1940
  3.  Story - https://bito.atlassian.net/browse/BITO-1914

Thanks,
Nisha
________________________________

**Comment by nisha-bito** [comment]
hi @MohammadMD1383 ,

JetBrains team has provided a workaround to solve this issue.
The fix should be be available with Bito's next release. We shall update you once we release the next build. 

Thanks,
Nisha

**Comment by Aleksey-Voko** [comment]
The same problem. In addition, entering a 6-digit verification code when logging into the plugin does not work.

Linux Mint 21.1
Bito AI 1.0.133

IntelliJ IDEA 2023.1 (Ultimate Edition)
PyCharm 2023.1 (Professional Edition)
WebStorm 2023.1

**Comment by bitoandy** [comment]
Hi Aleksey,

We are working on fix as Nisha mentioned JetBrains team has provided a workaround to solve this issue.
The fix should be available with Bito's next release. We shall update you once we release the next build.

________________________________

**Comment by Aleksey-Voko** [comment]
After the update, everything works fine.
Thanks.

**Comment by nisha-bito** [comment]
hi @MohammadMD1383 ,

We hope the reported issue is resolved for you as well with Bito version 1.0.134. 
In case you see any further issue, please feel free to reach out to us.

Thanks,
Nisha

**Comment by MohammadMD1383** [comment]
thanks!
